pvlib/pvsystem.py

```diff
@@ -2845,6 +2845,80 @@ def pvwatts_dc(g_poa_effective, temp_cell, pdc0, gamma_pdc, temp_ref=25.):
     return pdc
 
 
+def _infer_k_huld(cell_type):
+    # from PVGIS documentation, Section 5.2.3, accessed 12/22/2023
+    huld_params = {'cSi': (-0.017237, -0.040465, -0.004702, 0.000149,
+                           0.000170, 0.000005),
+                   'CIS': (-0.005554, -0.038724, -0.003723, -0.000905,
+                           -0.001256, 0.000001),
+                   'CdTe': (-0.046689, -0.072844, -0.002262, 0.000276,
+                            -0.000159, -0.000006)}
+    return huld_params[cell_type]
+
+
+def huld(effective_irradiance, temp_mod, pdc0, k=None, cell_type=None):
+    r"""
+    Power (DC) using the Huld model.
+
+    The Huld model [1]_ is given by
+
+
+    .. math::
+
+        P_{dc} = G` ( P_{dc0} k_1 log(G') + k_2 log^2 (G') + k_3 T' log(G') +
+                 k_5 T' log^2 (G') + k_6 T'^2)
+
+        G' = \frac{G_{poa eff}}{1000}
+
+        T' = T_{mod} - 25 \degree C
+
+
+    Parameters
+    ----------
+    effective_irradiance : numeric
+        The irradiance that is converted to photocurrent. [W/m^2]
+    temp_mod: numeric
+        Module back-surface temperature [C].
+    pdc0: numeric
+        Power of the modules at 1000 W/m^2 and cell reference temperature. [W]
+    k : tuple, optional
+        Length 6. If `k` is not provided, `cell_type` must be specified.
+    cell_type : str, optional
+        If provided, must be one of `'cSi'`, `'CIS'`, or `'CdTe'`. Used to look
+        up default values for `k` if `k` is not specified.
+
+    Returns
+    -------
+    pdc: numeric
+        DC power. [W]
+
+    Raises
+    ------
+    ValueError
+        If neither `k` nor `cell_type` are specified.
+
+    References
+    ----------
+    .. [1] T. Huld, G. Friesen, A. Skoczek, K. R. Kenny, T. Sample, M. Field,
+           E. Dunlop. A power-rating model for crystalline silicon PV modules.
+           Solar Energy Materials and Solar Cells 95, (2011), pp. 3359-3369.
+           10.1016/j.solmat.2011.07.026.
```

Comment:
```suggestion
           :doi:`10.1016/j.solmat.2011.07.026`.
```
I think this is the correct formatting of DOIs